sentinel/test/mocha/transitions/update_notifications.js

```diff
@@ -0,0 +1,355 @@
+const sinon = require('sinon').sandbox.create(),
+      transition = require('../../../transitions/update_notifications'),
+      db = require('../../../db-nano'),
+      utils = require('../../../lib/utils');
+
+describe('update_notifications', () => {
+
+  afterEach(done => {
+    sinon.restore();
+    done();
```

Comment:
Don't think the `before`/`after` hooks need the callback called (unlike nodeunit)